projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/profile-tag.ts

```diff
@@ -0,0 +1,135 @@
+export const ProfileTagHelper = {
```

Comment:
is there a reason to split all the helpers functions in different files. 

If yes, are other cds related e2e test will be using them?
AFAIK, this is the last e2e test.

If no, and it will only be used by `profile-tag`, then I believe you can put everything in this file

Reply:
Here was my thought process behind this:
- A combined file would be 500 lines long
- The profiletag helper would be updated by a different team than the merchandising-carousel helper, helping to avoid merge conflicts
- Both of these files will likely grow in the future

Reply:
that's fine